framework/proto/flwr/proto/control.proto

```diff
@@ -110,3 +114,6 @@ message ListNodesResponse {
   repeated NodeInfo nodes_info = 1;
   string now = 2;
 }
+
+message ListFederationsRequest {}
+message ListFederationsResponse { repeated string federations = 1; }
```

Comment:
Instead of `string`, should we perhaps return `Federation` messages, even if they just have a single attribute `name`?